requirements.txt

```diff
@@ -0,0 +1,50 @@
+astroid==2.4.2
+attrs==20.3.0
+blis==0.7.4
+catalogue==2.0.1
+certifi==2020.12.5
+chardet==4.0.0
+click==7.1.2
+cymem==2.0.5
+en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0-py3-none-any.whl
```

Comment:
Is this the only method of downloading spacy models? Isn't there some download script we could run in the Dockerfile, like for polyglot?

If that's the case, then you want to abandon Pipfile, and just concatenate the `requirements.txt` with more lines generated from a FOR loop, before running `pip install -r` on it.